src/NexusMods.App.UI/Initializers.cs

```diff
@@ -18,7 +18,7 @@ namespace NexusMods.App.UI;
 public static class Initializers
 {
     public static readonly ICommand ICommand = ReactiveCommand.Create(() => { });
-    public static readonly IImage IImage = new WriteableBitmap(new PixelSize(16, 16), new Vector(96, 96), PixelFormat.Rgba8888, AlphaFormat.Opaque);
+    public static IImage IImage => new WriteableBitmap(new PixelSize(16, 16), new Vector(96, 96), PixelFormat.Rgba8888, AlphaFormat.Opaque);
```

Comment:
This can't run until Avalonia's DI is setup, so moving it to a computed property fixes that race condition